api/openapi_server/controllers/auth_controller.py

```diff
@@ -150,6 +153,9 @@ def signUpCoordinator():  # noqa: E501
           SecretHash=secret_hash,
           Username=body['email'],
           Password=body['password'],
+          ClientMetadata={
+              'url': ROOT_URL
+          }
         )
     except Exception as e:
```

Comment:
We should fix this exception handler. 

The python `Exception` type does not have a `response` field/property. We are catching all types of exceptions, but treating it as if it has a `response` field. For example, if a `botocore.exceptions.ParamValidationError` is thrown this this except block will raise a different error.

Reply:
@Joshua-Douglas pushed some updated error handling. Let me know what you think!

Reply:
Looks great!